lib/otp/otp_db.c

```diff
@@ -39,7 +39,7 @@ RCSID("$Id$");
 #include "otp_locl.h"
 
 #if defined(HAVE_DB_NDBM)
-# include <ndbm.h>
+# include <db.h>
```

Comment:
Please test the following proposal:

Instead of this change, remove the entire block from line 41 to 45.  Then modify lib/roken/xdbm.h to add 
```

diff --git a/lib/roken/xdbm.h b/lib/roken/xdbm.h
index c2c6c28b4..1648be7da 100644
--- a/lib/roken/xdbm.h
+++ b/lib/roken/xdbm.h
@@ -47,6 +47,8 @@
 #elif defined(HAVE_NDBM_H)
 #include <ndbm.h>
 #endif
+#else /* !HAVE_NDBM */
+#include "ndbm_wrap.h"
 #endif /* HAVE_NDBM */
 
 #endif /* __XDBM_H__ */
```

otp_locl.h includes xdbm.h.

Reply:
This worked. Thanks.